example/heatEquation2D/src/analyticalSolution.hpp

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2020 Tapish Narwal
+ * SPDX-License-Identifier: ISC
+ */
+
+#pragma once
+
+#include <alpaka/alpaka.hpp>
+
+#include <cmath>
+
+//! Exact solution to the test problem
+//! u_t(x, y, t) = u_xx(x, t) + u_yy(y, t), x in [0, 1], y in [0, 1], t in [0, T]
+//! u(0, t) = u(1, t) = 0
+//! u(x, 0) = sin(pi * x)
+//! u(0, y) = sin(pi * y)
```

Comment:
I think initial and boundary conditions are not clear  in this problem definition.

Reply:
![Screenshot from 2024-08-20 13-47-25](https://github.com/user-attachments/assets/4fa0a3b3-40d1-45da-8f4b-2b1fe57e8177)

Reply:
We do not have Dirichlet or Neumann boundaries. We have an analytical solution to the equation with a special type of boundary condition not inspired from physics. This example is chosen because it is easy to verify the correctness of the finite difference stencil kernel and not because the physics is of any particular interest.
I have removed the outdated boundaries copied over from the 1D example

Reply:
Yes I see ( `u(0, t) = u(1, t) = 0` can not be initial condition for a 2D problem. )